### PR TITLE
feat: restake tcl positions to aura

### PR DESCRIPTION
**Pull request by gosuto-inzasheru**
solution to #1063

includes a call to script which solves #1048

### CODE REVIEW

**Review by jalbrekt85 — COMMENTED** [review]
note: this line has to be commented out to fully test sim:
https://github.com/Badger-Finance/badger-multisig/blob/fd8860f6f57e6a21927075b1fabb564224b26231/scripts/issue/1048/double_down_on_badgerreth.py#L74

File: scripts/issue/1063/restake_tcl_aura.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+    vault.take_snapshot(bpts + new_aura_gauges)
+
+    for gauge in old_balancer_gauges:
+        gauge.withdraw(gauge.balanceOf(vault))
+
+    vault.print_snapshot()
+
+    # NOTE: i commented the `post_safe_tx` out of this script:
+    run_script("scripts/issue/1048/double_down_on_badgerreth.py")
```
Comment (jalbrekt85) [inline scripts/issue/1063/restake_tcl_aura.py:34]: wasn't aware of `run_script`, nice. kinda surprised that this works though, i'd assume using a different instance of `GreatApeSafe` for `vault` in the badger/reth deposit would mean that the tx would not carry over into this script when it is ultimately posted from a different instance. but looking at the final snapshot and `history`, everything is included.

**Review by petrovska-petro — APPROVED** [review]
Withdraw from old gauges successfully, cross-check addr with [tx](https://etherscan.io/tx/0xf59f81f2816b02ff238aafc64871e36dcca70484930587febdda62b1927b4ad9), increases tcl in rethbadger pool leaving unstake the bpts and stakes on the latest reward contract within the aura infra, checked pid targeted (11, 19 & 18) all good ✅

**Review by jalbrekt85 — APPROVED** [review]
balancer withdrawal snap:
```
snapshot result for 0xD0A7A8B98957b9CD3cFB9c0425AbE44551158e9e:
                                 balance_before               balance_after               balance_delta
symbol                                                                                                 
20WBTC-80BADGER            0.000000000000000000  173,465.699734216429542964  173,465.699734216429542964
50BADGER-50rETH            0.000000000000000000   26,519.940569706080741029   26,519.940569706080741029
40WBTC-40DIGG-20graviAURA  0.000000000000000000      566.388828707154216788      566.388828707154216788 
```

badger/reth deposit:
```
rETH                    12.553624153997098192         17.260744510653908733         4.707120356656810541
WETH                   941.699344603880876279          0.000000000000000000      -941.699344603880876279
BADGER           2,917,362.747812842638976895  2,406,932.761807414530454049  -510,429.986005428108522846
50BADGER-50rETH     26,519.940569706080741029     68,137.532959102368498272    41,617.592389396287757243 
```

final snap after aura deposit:
```
snapshot result for 0xD0A7A8B98957b9CD3cFB9c0425AbE44551158e9e:
                                           balance_before               balance_after               balance_delta
symbol                                                                                                           
aura20WBTC-80BADGER-vault            0.000000000000000000  173,465.699734216429542964  173,465.699734216429542964
aura40WBTC-40DIGG-20graviAURA-vault  0.000000000000000000      566.388828707154216788      566.388828707154216788
aura50BADGER-50rETH-vault            0.000000000000000000   68,137.532959102368498272   68,137.532959102368498272 
```

lgtm

File: scripts/issue/1063/restake_tcl_aura.py (hunk 1 above)
Comment (gosuto-inzasheru) [reply scripts/issue/1063/restake_tcl_aura.py:34]: i thought this as well at first, but ultimately the tx is indeed built from receipts in `history`, which apparently is completely independent of the `GreatApeSafe` object.

Comment (gosuto-inzasheru) [reply scripts/issue/1063/restake_tcl_aura.py:34]: https://eth-brownie.readthedocs.io/en/stable/api-project.html#scripts.run

**Pull request merged**